src/engine/controls/cuecontrol.cpp

```diff
@@ -356,6 +356,7 @@ void CueControl::trackLoaded(TrackPointer pNewTrack) {
         m_pOutroEndEnabled->forceSet(0.0);
         m_pHotcueFocus->set(Cue::kNoHotCue);
         m_pLoadedTrack.reset();
+        m_oldSeekOnLoadPosition.setValue(0.0);
```

Comment:
kDefaultSeekOnLoadPosition?

Reply:
This is more like the uninitialized value "kNotSeekedOnLoadPosition". This should fit to the new name for the variable we decide for. Ideas?

Reply:
No need to argue about names. But we use 0.0 too often with lots of implicit meanings.